pkg/templates/installation_context.go

```diff
@@ -35,6 +35,22 @@ func (ctx *InstallationContext) entitlementValue(name string) string {
 }
 
 func (ctx *InstallationContext) shipCustomerRelease() string {
+	restrictedMeta := ctx.Meta
+	restrictedMeta.ConfigSpec = ""
```

Comment:
it would be nice to omit these but this seems like a good workaround for the time being. although it would be possible to marshal it to a map string interface and remove the keys.